prow/config.yaml

```diff
@@ -178,6 +183,9 @@ presubmits:
       - name: service
         secret:
           secretName: service-account
+      - name: coveralls
```

Comment:
I don't think you need this anymore since it isn't being mounted as a file

Reply:
okay, done. I think this is the code we want now?

Reply:
Yes, just `/hold cancel` once you're ready to merge